scenarios/command/unlock_command.proto

```diff
@@ -0,0 +1,17 @@
+﻿syntax = "proto3";
```

Comment:
@rido-min does it make sense to move this file to the root of the command folder since it's shared across the languages? It broke the dotnet samples, so I couldn't remove the other copy

Reply:
I think it's ok to duplicate the proto files per language, since each one might require different tools.